### ISSUE TITLE
Some toggle buttons, unlike others, don't allow two lines of text

### ISSUE THREAD

**Issue by kitsunenosaraT** [mozilla-mobile/firefox-ios#6416]
## Steps to reproduce
Compare screenshots [iPhone 8-TrackingProtectionStrictWarning-01.png](https://drive.google.com/file/d/1bCb2uA-x7IWRHRzwoDADmVtGoEgF47tn/view) and [iPhone 8-22TrackingProtectionBlockedElements-01.png](https://drive.google.com/file/d/1IqkkiSarhR_NlV1IosZQ2vtNJGRA380f/view) (locale is Italian).

### Expected behavior
In _iPhone 8~~22TrackingProtectionBlockedElements~~01.png_ the text of the toggle button should be accommodated on two lines and fully displayed, as it happens for _iPhone 8~~TrackingProtectionStrictWarning~~01.png_.

### Actual behavior
In _iPhone 8~~22TrackingProtectionBlockedElements~~01.png_ the text of the toggle button appears truncated with final ellipses.

### Device & build information
* Device: ?
* Build version: 25.0 (17610)

### Notes
Attachments: 
![](https://user-images.githubusercontent.com/14000019/79443107-b33d6f00-7fd9-11ea-9a38-dc72fbe82391.png)
![](https://user-images.githubusercontent.com/14000019/79443109-b46e9c00-7fd9-11ea-9b13-a6fa641fa48c.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2349)

**Comment by nbhasin2** [comment]
- This should go to two lines if possible
- Tech Debt:  Also look into making the sheet dynamic to support both truncated and multi line

**Comment by data-sync-user** [comment]
➤ Catalin Suciu commented:

Verifying as fixed on v9000 (24964)
